instagram/src/components/PostContainer/PostContainer.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Post from './Post';
+import PropTypes from 'prop-types';
+import "./PostContainer.css"
+
+
+function PostContainer(props){
+    return(
+        <div className="post-container">
+        {
+            props.post.map( e => (
+                // console.log(e.display)
+                <div className="post" key={e.index} style={{display:`${e.display}`}}> 
+                <Post 
+                    username={e.username} 
```

Comment:
You should definitely just send down your entire object `e` here and destructure it from the context of `Post.js`